src/kit/Divider.tsx

```diff
@@ -0,0 +1,11 @@
+import { Divider as AntdDivider } from 'antd';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  children?: ReactNode;
+}
+const Divider: React.FC<Props> = ({ children }: Props) => {
+  return <AntdDivider>{children}</AntdDivider>;
```

Comment:
nit: by default the divider has 24px of margin on top and bottom which seems excessive. I guess this is an @alexjdetermined question, but maybe we could lower it to like 12px?